about.html

```diff
@@ -81,57 +87,63 @@ <h2>South Korea</h2>
         <!--div 3 light-->
         <div class="bg-light text-dark p-2">
             <h2 class="text-center">See the past Developing Contest Global winners</h2>
-            <hr class="my-4 text-primary bg-primary border-5">
+            <hr>
             <p class="text-center"> Take a look at the last two developing contest global winner, celebrated in Rio do Janeiro and in Vienna</p>
             
-            <div class="card text-center p-2 hero-image text-light mb-2 tw-400">
-                <div class="card-body">
-                  <h3 class="card-title">2019</h3>
-                  <h6 class="card-subtitle mb-2">Winner: Japan</h6>
-                  <p class="card-text">Location: Brazil</p>
+            <div class="presenters">
+                <div class="card text-center hero-image text-light mb-2 tw-400">
+                    <div class="card-body hero-div">
+                      <h3 class="card-title mt-2">2019</h3>
+                      <h6 class="card-subtitle mb-2">Winner: Japan</h6>
```

Comment:
- [ ] I'm sure on the images, you could add a space between them (on desktop size) and have it more centered. Please work on this. 🐔 
![image](https://user-images.githubusercontent.com/20563252/132961619-182f63f2-18f3-4001-be7f-5890c59b395c.png)